app/build.gradle

```diff
@@ -23,15 +32,66 @@ android {
         }
     }
 
+    kotlinOptions {
+        jvmTarget = JavaVersion.VERSION_1_8.toString()
+    }
+
 }
 
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
+
+    // Kotlin
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
-    implementation 'androidx.appcompat:appcompat:1.1.0'
-    implementation 'androidx.core:core-ktx:1.2.0'
+    implementation "org.jetbrains.kotlin:kotlin-reflect:$kotlin_version"
+
+    // Constraint Layout
     implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
-    testImplementation 'junit:junit:4.12'
+
+    // Core with ktx
+    implementation 'androidx.core:core-ktx:1.3.0'
+
+    // Support
+    implementation 'androidx.appcompat:appcompat:1.1.0'
+    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
+
+    // Material Design
+    implementation 'com.google.android.material:material:1.2.0-beta01'
```

Comment:
Avoid using beta version, unless there's a feature you really need in the beta version